sdk/src/main/java/io/customer/sdk/data/request/Metric.kt

```diff
@@ -3,6 +3,7 @@ package io.customer.sdk.data.request
 import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
+@JsonClass(generateAdapter = false)
```

Comment:
🤔 I have only used `generateAdapter = true`. Why are we using `false` here?

Reply:
R8/Pro-guard removes them, hence this is [recommended](https://github.com/square/moshi#r8--proguard) by Moshi. 

Moshi don't need adapters for Enums, it caters them internally.

Reply:
Ah, 🤦‍♂️. I saw your link (thank you) in the description but didn't see the section titled "Enum". Got it.